internal/layers/layers.go

```diff
@@ -95,22 +101,21 @@ func MountLCOWLayers(ctx context.Context, containerID string, layerFolders []str
 
 	for _, layerPath := range layerFolders[:len(layerFolders)-1] {
 		log.G(ctx).WithField("layerPath", layerPath).Debug("mounting layer")
-		var (
+		if path.Ext(layerPath) != ".vhd" && path.Ext(layerPath) != ".vhdx" {
 			layerPath = filepath.Join(layerPath, "layer.vhd")
-			uvmPath   string
-		)
-		uvmPath, err = addLCOWLayer(ctx, vm, layerPath)
+		}
```

Comment:
This seems a bit hacky. Can we find a better way to pass down information on the layer path?

Reply:
Are you referring to the file path extension?

Reply:
Yes, specifically depending on the extension to change behavior

Reply:
It's not meant to really change the behavior, just to normalize the input we get to `addLCOWLayer`.